doc/modules/mixture.rst

```diff
@@ -195,6 +168,44 @@ to some mixture components getting almost all the points while most
 mixture components will be centered on just a few of the remaining
 points.
 
+.. figure:: ../auto_examples/mixture/images/plot_bayesian_gaussian_mixture_001.png
+   :target: ../auto_examples/mixture/plot_bayesian_gaussian_mixture.html
+   :align: center
+   :scale: 50%
+
+.. topic:: Examples:
+
+    * See :ref:`plot_bayesian_gaussian_mixture.py` for a comparaison of
+      the results of the ``BayesianGaussianMixture`` for different values
+      of the parameter ``alpha``.
+
+Pros and cons of class :class:`BayesianGaussianMixture`: variational inference
+------------------------------------------------------------------------------
+
+Pros
+.....
+
+:Regularization: due to the incorporation of prior information,
+   variational solutions have less pathological special cases than
+   expectation-maximization solutions. One can then use full
+   covariance matrices in high dimensions or in cases where some
+   components might be centered around a single point without
+   risking divergence.
+
```

Comment:
I would also had that the Bayesian mixture model has a natural tendency to set some mixture weights to values close to zero when `alpha` is small enough and `n_components` is larger that what is found necessary by the model. This makes it possible to let the model choose a suitable number of effective components automatically.